microsoft-365-ops/3.0/modules/ROOT/pages/microsoft-365-finance-operations-connector-studio.adoc

```diff
@@ -0,0 +1,131 @@
+* <<oauth-client-credentials,*OAuth Client Credentials*>>
+* <<oauth-2.0,*OAuth 2.0*>>
++
+You can reference a configuration file that contains ANT-style property placeholders (recommended), or you can enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[oauth-client-credentials]]
+=== OAuth Client Credentials
+
+Enter the following required information on the *General* tab of the *Global Element Properties* screen to configure *OAuth Client Credentials* authentication:
+
+[%header,cols="30s,70a"]
+|===
+|Field |Description
+| Resource a| App ID URI of the web API
+| Client Id a| OAuth client ID as registered with the service provider. 
+| Client Secret a| OAuth client secret as registered with the service provider
+|===
+
+
+
+
+[[oauth-2.0]]
+=== OAuth 2.0
+
+Enter the following information on the *General* tab of the *Global Element Properties* screen to configure *OAuth 2.0* authentication:
+
+[%header,cols="30s,70a"]
+|===
+|Field |Description
+|Resource |The OAuth endpoint URL.
+|Consumer Key |The app ID assigned to your app when you registered it with Azure Active Directory. You can find this in the Azure Portal (portal.azure.com). Click *Active Directory*, click the directory, choose the app, and click *Configure*.
+|Consumer Secret |The secret that you created in the app registration portal for your app. Do not use this in a native app because client secrets cannot be reliably stored on devices. This is required for web apps and web APIs, which have the ability to store the client secret securely on the server side. You can find this in the Azure Portal (portal.azure.com). A client secret is a numerical value known only to your app and the Azure server, which enables access for your app.
+|Authorization URL |Tenant-specific OAuth2.0 endpoint in the format: `+https://login.microsoftonline.com/{tenant}/oauth2/authorize+` (follow instructions presented in the previous link for specific information on how to obtain the endpoint)
+|Access token URL |Tenant-specific endpoint for requesting the access token in the format: `+https://login.microsoftonline.com/{tenant}/oauth2/token+` (follow instructions presented in the previous link for specific information on how to obtain the endpoint).
+|Listener config |Name of the HTTP Listener instance created in the Anypoint Studio flow.
+|Callback path |The URL of your app, to which authentication responses can be sent and received by your app (follow instructions presented in the above link for specific information on how to obtain the endpoint).
```

Comment:
I am confused by the wording. It sounds like authentication responses can be sent by your app to this URL, which isn't true. Maybe say something like this (which is not great either:

The URL of your app. Your app can send authentication responses from this URL and receive them at this URL.